File: Day 2/JavaScript/Level 2/TaskFive.js
const prompt = require('prompt-sync')();

let numberOne = Number(prompt("Enter number: "));
let numberTwo = Number(prompt("Enter number: "));
let numberThree = Number(prompt("Enter number: "));

let largestNumber = numberOne;

if (largestNumber < numberTwo) largestNumber = numberTwo;
else if (largestNumber < numberThree) largestNumber = numberThree;

console.log("Largest number is", largestNumber);

File: Day 2/Java/Level 2/TaskFive.java
import java.util.Scanner;

public class TaskFour{
    public static void main(String[] args){
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter first number: ");
        double numberOne = scanner.nextDouble();
        System.out.print("Enter second number: ");
        double numberTwo = scanner.nextDouble();
        System.out.print("Enter third number: ");
        double numberThree = scanner.nextDouble();

        double largestNumber = numberOne;

        if (largestNumber < numberThree) largestNumber = numberTwo;
        else if (largestNumber < numberThree) largestNumber = numberThree;
        
        System.out.print("Largest number is %f%n", largestNumber);
    }
}
